### PR TITLE
Upgrade plugin-publish to version 0.13.0

### PR DESCRIPTION
**Pull request by pbielicki**
Use new feature of publish-plugin to inject plugin portal credentials via environment variables.

### CODE REVIEW

File: .teamcity/settings/extensions.kt [hunk 1]
```diff
@@ -50,6 +51,8 @@ fun Project.buildType(buildTypeName: String, init: BuildType.() -> Unit): BuildT
             param("env.GRADLE_CACHE_REMOTE_URL", "%gradle.cache.remote.url%")
             param("env.GRADLE_CACHE_REMOTE_USERNAME", "%gradle.cache.remote.username%")
             param("env.GRADLE_CACHE_REMOTE_PASSWORD", "%gradle.cache.remote.password%")
+            param("env.GRADLE_PUBLISH_KEY", "%%pluginPortalPublishKey%%")
+            password("env.GRADLE_PUBLISH_SECRET", "%%pluginPortalPublishSecret%", display = NORMAL)
```
Comment (britter) [inline .teamcity/settings/extensions.kt:55]:
```suggestion
            param("env.GRADLE_PUBLISH_KEY", "%pluginPortalPublishKey%")
            password("env.GRADLE_PUBLISH_SECRET", "%pluginPortalPublishSecret%", display = NORMAL)
```

File: .teamcity/settings.kts [hunk 2]
```diff
@@ -168,23 +173,25 @@ project {
             params {
                 select(
                     "releaseScope", "", label = "releaseScope", description = "The scope of the release",
-                    display = ParameterDisplay.PROMPT, options = listOf("major", "minor", "patch")
+                    display = PROMPT, options = listOf("major", "minor", "patch")
                 )
                 text(
-                    "githubUsername",
+                    "env.GRGIT_USER",
```
Comment (jonnybbb) [inline .teamcity/settings.kts:179]: ❓ Can you explain to me what this is supposed to do? Are those `GRGIT_*` internal (undocumented?) environment variables?

**Review by jonnybbb — APPROVED** [review]
Great that you also tackled the github credentials ㊙️

File: .teamcity/settings.kts (hunk 2 above)
Comment (pbielicki) [reply .teamcity/settings.kts:179]: nebula plugin (which we use directly) uses grgit plugin which then tags Git and needs these credentials - see http://ajoberstar.org/grgit/grgit-authentication.html

**Review by britter — APPROVED** [review]

**Pull request merged**